test/runtests.jl

```diff
@@ -43,26 +53,26 @@ end
     @test all(sv .== [3, 5, 4, 1])
     diff = v .- sv
     @test diff == [-2, -2, 1, 3]
-    @test shifts(sv) == (-1,)
+    @test shifts(sv) == (3,)
     sv2 = CircShiftedVector(v, 1)
     diff = v .- sv2
     @test copy(sv2) == [4, 1, 3, 5]
     @test all(CircShiftedVector(v, 1) .== circshift(v,1))
     sv[2] = 0
     @test collect(sv) == [3, 0, 4, 1]
     @test v == [1, 3, 0, 4]
-    sv[7] = 12
+    sv[3] = 12 
```

Comment:
```suggestion
    sv[3] = 12
```
(trailing whitespace)